pyfolio/timeseries.py

```diff
@@ -491,95 +493,129 @@ def stability_of_timeseries(returns):
 
     """
 
-    if returns.size < 2:
-        return np.nan
+    cum_log_returns = np.log(1 + returns).cumsum()
+    rhat = stats.linregress(np.arange(len(cum_log_returns)),
+                            cum_log_returns.values)[2]
 
-    df_cum_rets = cum_returns(returns, starting_value=100)
-    df_cum_rets_log = np.log10(df_cum_rets.values)
-    len_returns = df_cum_rets.size
+    return rhat
 
-    X = list(range(0, len_returns))
-    X = sm.add_constant(X)
 
-    model = sm.OLS(df_cum_rets_log, X).fit()
+SIMPLE_STAT_FUNCS = [
```

Comment:
Do we intend on mutating these later? I would prefer an immutable structure for these module constants unless it is an intended feature that we can mutate them.